kolibri/utils/cli.py

```diff
@@ -19,9 +19,14 @@
     os.path.realpath(os.path.dirname(kolibri_dist.__file__))
 ]
 
+try:
+    from build_config.default_settings import settings_path
```

Comment:
My main worry with the PR, is that there might be too many layers of indirection happening. That might make debugging harder in production, as now we need to watch out for `build_config`, an auto-generated file, in addition to the environment variable. Is there a way we can reduce these indirections? Maybe we can always inherit from a single file `kolibri.deployment.settings`, which imports `kolibri.deployment.default.settings.base`, but can be appended to during build time for other settings? Not the best solution, but maybe we can iterate from here.

Is there a way we can reduce that here? Maybe we can just always write to a known settings

Reply:
I see the concern, but the point here is not to create a new settings file, but rather store the settings path somewhere.

Perhaps this could be mitigated by logging the settings path that we are using during server startup? That way the log file would always unambigously specify which settings file was actually being used.

This would be important because even with this provision, all this is doing is changing the default `DJANGO_SETTINGS_MODULE` it can still be changed by an external env var at run time.